src/main.rs

```diff
@@ -210,13 +210,14 @@ async fn entry() -> Result<()> {
     // Fetch crate via crates.io, git, or use a local manifest path
     // TODO: work out which of these to do based on `opts.name`
     // TODO: support git-based fetches (whole repo name rather than just crate name)
-    let manifest_path = match opts.manifest_path.clone() {
-        Some(p) => p,
-        None => fetch_crate_cratesio(&opts.name, &opts.version, temp_dir.path()).await?,
+    let manifest = match opts.manifest_path.clone() {
+        Some(manifest_path) => {
+            debug!("Reading manifest: {}", manifest_path.display());
```

Comment:
Let's put that call in the fn. Also will be more correct as we're really reading the `Cargo.toml`

Reply:
Turn out that [`load_manifest_path`] is already doing that:

```
    debug!("Reading manifest: {}", manifest_path.as_ref().display());
```

So I will just remove this line.

[`load_manifest_path`]: https://github.com/ryankurte/cargo-binstall/blob/c33f195d5fad6b7927f9829d427729614667800d/src/helpers.rs#L30